src/Microsoft.ML.Data/Transforms/FeatureContributionCalculationTransformer.cs

```diff
@@ -266,9 +227,68 @@ private Delegate GetValueGetter<TSrc>(DataViewRow input, int colSrc)
     }
 
     /// <summary>
-    /// Estimator producing a FeatureContributionCalculatingTransformer which scores the model on an input dataset and
-    /// computes model-specific contribution scores for each feature.
+    /// Computes model-specific per-feature contributions to the score of each input vector.
+    /// See the list of currently supported models below.
```

Comment:
>See the list of currently supported models below. [](start = 8, length = 49)

I would remove this, because this line displays on the IntelliSense, and there won't be an option to see the remarks section there.